content/.well-known/security.txt

```diff
@@ -0,0 +1,7 @@
+Contact: mailto:security@matrix.org
+Expires: 2022-01-30T23:00:00.000Z
```

Comment:
```suggestion
Expires: 2024-01-30T23:00:00.000Z
```

I'm assuming this shouldn't already be expired. Could do 2023 but there are only a few months left until 2023.

Reply:
Oops it should have been 2023 mot 2022 :) the rfc says its recommended to be less than 1 year to ensure it not getting stale. Should I go with 2024 anyway? :)

Reply:
None of the information in here has changed since 2019 so imo there isn't much chance of it getting stale.

Reply:
ok :) Then I just apply your suggestion 👍